includes/admin/class-wcmp-export.php

```diff
@@ -1033,6 +1035,27 @@ public static function getPackageTypeAsString($packageType): string
         return $packageType ?? AbstractConsignment::DEFAULT_PACKAGE_TYPE_NAME;
     }
 
+    /**
+     * @param WC_Order $order
+     * @param string   $packageType
+     *
+     * @return string
+     *
+     * @throws Exception
+     */
+    public function getAllowedPackageType(WC_Order $order, string $packageType): string
+    {
+        $shipping_country = WCX_Order::get_prop($order, "shipping_country");
+        $mailbox          = AbstractConsignment::PACKAGE_TYPE_MAILBOX_NAME === $packageType;
```

Comment:
Deze twee renamen naar `is<packageType>` is leesbaarder™️

Reply:
done